benches/coprocessor_executors/hash_aggr/util.rs

```diff
@@ -63,7 +64,7 @@ impl HashAggrBencher for NormalBencher {
                 black_box(Box::new(src)),
             )
             .unwrap();
-            Box::new(ex) as Box<dyn Executor>
+            Box::new(ex) as Box<dyn Executor<StorageStats = Statistics>>
```

Comment:
In the benches, can we use `()` as the stat type? It is real pain to compile benches. We don't want to depend on the tikv trait to run benches.

Reply:
Good suggestion. I will refine it.

Reply:
Considering that this benchmark itself also benches *real RocksDB scenario*, would you accept a feature gate solution?

Reply:
I find it easy to extract the RocksDB engine out.
(Actually, I find `storage::Statistics` seems easy to be extracted out too...If we extract storage statistics, it seems not so useful that we split coprocessor execution and storage statistics...)

Reply:
@sticnarf As a standalone query executor, it should not depend on specific storage statistics.

Reply:
Makes sense.